src/config/db.config.ts

```diff
@@ -20,6 +21,7 @@ if (process.env.VCAP_SERVICES) {
   pgUser = svcCredentials.username;
   pgPwd = svcCredentials.password;
   pgDb = svcCredentials.name;
+  pgReplicaHost = svcCredentials.replica_host || process.env.EASEY_DB_REPLICA_HOST || pgHost;
```

Comment:
Is there a use case for falling back to the `EASEY_DB_REPLICA_HOST` value? For consistency with the other assignments in this block, it may be better to stick to only the values in the `svcCredentials`.

```suggestion
  pgReplicaHost = svcCredentials.replica_host || pgHost;
```